sdk/communication/Azure.Communication.NetworkTraversal/api/Azure.Communication.NetworkTraversal.netstandard2.0.cs

```diff
@@ -0,0 +1,34 @@
+namespace Azure.Communication.NetworkTraversal
+{
+    public partial class CommunicationRelayClient
+    {
+        protected CommunicationRelayClient() { }
+        public CommunicationRelayClient(string connectionString) { }
+        public CommunicationRelayClient(string connectionString, Azure.Communication.NetworkTraversal.CommunicationRelayClientOptions options) { }
+        public CommunicationRelayClient(System.Uri endpoint, Azure.AzureKeyCredential keyCredential, Azure.Communication.NetworkTraversal.CommunicationRelayClientOptions options = null) { }
+        public CommunicationRelayClient(System.Uri endpoint, Azure.Core.TokenCredential tokenCredential, Azure.Communication.NetworkTraversal.CommunicationRelayClientOptions options = null) { }
+        public virtual Azure.Response<Azure.Communication.NetworkTraversal.CommunicationRelayConfiguration> GetRelayConfiguration(Azure.Communication.CommunicationUserIdentifier communicationUser, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
```

Comment:
I thought we are starting by a version that does not need a communication user and we create that on-demand on the server. Was I mistaken?

Reply:
> starting by a version that does not need a communication user

Nope, what's working today is a requirement for a communication user and it was agreed that there are issues with having the service implicitly create user identities without the Contoso Server being aware of it. A non-user endpoint is planned to be released when the internal infrastructure does not require a communication user